src/source/geojson_source.js

```diff
@@ -230,6 +230,22 @@ class GeoJSONSource extends Evented implements Source {
      * @param offset The number of features to skip (e.g. for pagination).
      * @param callback A callback to be called when the features are retrieved (`(error, features) => { ... }`).
      * @returns {GeoJSONSource} this
+     * @example
```

Comment:
per convo between @malwoodsantoro and @asheemmamoowala, we may also want to add a `@see` example here.

Reply:
I don't think we have any examples for this, unfortunately.